### ISSUE TITLE
'' as an attribute name in array

### ISSUE THREAD

**Issue by whereiswolf** [lukix/input-reader#1]
Consider the code below (Based on the "Pizza" challenge in HashCode's Judge sytem):  
```javascript
const reader = require('input-reader');
const fs = require('fs');

const data = fs.readFileSync('./in/a_example.in').toString();
const pattern = `
  rows columns minIngredients maxCells
  ...pizza[]
`;

console.log(reader(data, pattern));
```
will parse the data accordingly:
```javascript
{ 
  rows: 3,
  columns: 5,
  minIngredients: 1,
  maxCells: 6,
  pizza: [ { '': 'TTTTT' }, { '': 'TMMMT' }, { '': 'TTTTT' } ]
}
```
what requires using an additional `map` function.
If no attribute name is provided, shouldn't the array be parsed automatically to:
```javascript
{ 
  ...
  pizza: [ 'TTTTT', 'TMMMT', 'TTTTT' } ]
}
```
??

Thanks!

**Comment by lukix** [comment]
Sure. I'll implement that when I have some free time (probably beginning of February).

**Comment by lukix** [comment]
The issue has been addressed but in a slightly different way.
https://github.com/lukix/input-reader/pull/5/files

Output for your example will look like this:
```javascript
{ 
  rows: 3,
  columns: 5,
  minIngredients: 1,
  maxCells: 6,
  pizza: [ ['TTTTT'], ['TMMMT'], ['TTTTT'] ]
}
```
There is an additional nesting level which you don't need, but IMHO it was necessary and natural to make it work that way (array of rows and each row is an array of values). 
It has been published as version 2.0.0 (along with some bug fixes & unit tests).

Thanks for your engagement!
